Sources/AblyChat/Messages.swift

```diff
@@ -26,15 +51,17 @@ public struct QueryOptions: Sendable {
     }
 
     public var start: Date?
-    public var end: Date?
+    public var end: Int? // represented as timeserial
```

Comment:
Why the public API change?

Reply:
I've added a todo to solve the mapping to swifty types after

Reply:
Now `start` is a `Date?` and `end` is `Int?`. Should be fixed in this PR IMHO.

Reply:
Yeah, I agree with Marat, the inconsistency between `start` and `end` makes this even more confusing.

The part of the spec that describes the `GET /chat/v1/rooms/<roomId>/messages` API says "The method accepts query parameters identical to the standard Ably REST API." This is a bit vague, but I _assume_ that what it means is that it accepts the same parameters as the [REST history API](https://ably.com/docs/api/rest-api#message-history) (I suggest you ask Andy to clarify this in the spec). If that is the case, then `start` and `end` are not a "timeserial" as you've said here, they are (per the documentation) "a time in milliseconds since the epoch". 

For whatever reason, ably-js chose to use `number` to represent milliseconds since the epoch, and it seems like the JS Chat SDK just continued this convention. ably-cocoa, on the other hand, [chose to use `Date`](https://github.com/ably/ably-cocoa/blob/63e6f001d06cb7defb6be92f87a831f920eaf8c1/Source/include/Ably/ARTDataQuery.h#L19-L27), so shouldn't we do the same here?

I wouldn't mind this work being deferred for later if it were just "let's do the conversion from milliseconds to `Date` later" (although I wouldn't fully understand _why_ it needs to be deferred), but it seems like we need to clear up the _meaning_ of `end`, because I do not know what it means that it is "represented as timeserial", and even if I did I’m not convinced that it's correct.

Reply:
ahh [this](https://github.com/ably/ably-cocoa/blob/63e6f001d06cb7defb6be92f87a831f920eaf8c1/Source/include/Ably/ARTDataQuery.h#L19-L27) helps a lot... my misunderstanding. I'll use the same helpers from ably-cocoa